frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/confirmation.tsx

```diff
@@ -268,11 +269,10 @@ export default function ApplyFlowConfirm() {
     <div className="max-w-prose">
       <ContextualAlert type="success">
         <div className="space-y-4">
-          <h2 className="text-xl font-semibold">{t('confirm.alert-heading')}</h2>
-          <p>
-            {t('confirm.app-code-is')}
+          <p className="text-2xl">
+            <strong>{t('confirm.app-code-is')}</strong>
             <br />
-            <strong>{submissionInfo?.confirmationCode ?? 'confirmationCode'}</strong>
+            <strong>{formatSubmissionApplicationCode(submissionInfo?.confirmationCode ?? '1234567891011')}</strong>
```

Comment:
I'm assuming the defaulted confirmation code here is for testing purposes and/or the API isn't ready yet.

Reply:
correct, it's a bit of a dog's breakfast right now.